.eslintrc.json

```diff
@@ -27,7 +37,7 @@
     "eqeqeq": [2, "allow-null"],
     "generator-star-spacing": [2, { "before": true, "after": true }],
     "handle-callback-err": [2, "^(err|error)$" ],
-    "indent": [2, 2, { "SwitchCase": 1 }],
+    "indent": [2, "tab", { "SwitchCase": 1 }],
```

Comment:
What is the relevance of `2` and `{ "SwitchCase": 1 }`  instead of just `"tab"`?

Reply:
Cool.